src/coreclr/src/tools/Common/TypeSystem/Common/TypeDesc.cs

```diff
@@ -629,5 +629,17 @@ public bool IsByRefLike
                 return (GetTypeFlags(TypeFlags.IsByRefLike | TypeFlags.AttributeCacheComputed) & TypeFlags.IsByRefLike) != 0;
             }
         }
+
+        /// <summary>
+        /// Verifies that a type can be fully loaded successfully. If not, a TypeSystem exception will be thrown
```

Comment:
There is no concept of "fully loaded type" in the CoreRT type system today. Everything is done lazily.

What does it mean for the type to be fully loaded?

Reply:
I can rename this to ValidateType or something. I do agree it's not really about fully loading

Reply:
ValidateType has the same problem: What does it mean that the type is valid?

Reply:
I ended up using the existing functionality in the `CompileTypeSystemContext`